### PR TITLE
Add command to list routers

### PR DESCRIPTION
**Pull request by lucasweiblen**

### CODE REVIEW

File: plan.go [hunk 1]
```diff
@@ -115,3 +116,42 @@ func (c *planRemove) Run(context *cmd.Context, client *cmd.Client) error {
...
+	return &cmd.Info{
+		Name:    "router-list",
+		Usage:   "router-list",
+		Desc:    "List all routers available for plan creation.",
+		MinArgs: 0,
+	}
+}
+
+func (c *planRoutersList) Run(context *cmd.Context, client *cmd.Client) error {
+	url, err := cmd.GetURL("/plan/routers")
```
Comment (fsouza) [inline plan.go]: It should be ``/plans/routers``.

I was wondering whether we could add some integration tests to our clients to detect these typos in the URL or the method.

**Comment by fsouza** [comment]
Hi @lucasweiblen, looks good so far! Just the URL fix and I think we can merge it.

Thank you!

File: plan.go (hunk 1 above)
Comment (lucasweiblen) [inline plan.go]: good catch! my bad :)

**Comment by fsouza** [comment]
Can you fix the build? Then we're good to merge. Thank you!

**Comment by lucasweiblen** [comment]
I'm trying to understand why the build is breaking.

Logs says the following:
./plan.go:144: undefined: router.PlanRouter
./plan_test.go:182: undefined: router.PlanRouter
./plan_test.go:183: undefined: router.PlanRouter
./plan_test.go:184: undefined: router.PlanRouter

But this is already on master. Am I missing something here?

**Comment by fsouza** [comment]
@lucasweiblen we use Godeps to pin dependencies, so you need to update the reference of tsuru that you're using in tsuru-admin:

```
% cd $GOPATH/src/github.com/tsuru/tsuru-admin
% godep update github.com/tsuru/tsuru/...
% git commit Godeps -m "Godeps: update tsuru"
```

**Comment by lucasweiblen** [comment]
@fsouza hm... now I got some "fatal: reference is not a tree" errors.. any idea what's this?

**Comment by fsouza** [comment]
No :-( Is tsuru cloned and checked out at master in the directory $GOPATH/src/github.com/tsuru/tsuru?

**Comment by lucasweiblen** [comment]
I pulled the last changes and checkout at master on Tsuru and updated godeps.. let's see if it works now...

**Comment by fsouza** [comment]
Seems like it reverted some changes in go-dockerclient and docker-cluster versions. It should be updating just github.com/tsuru/tsuru/..., leaving other packages alone. Did you run ``godep update github.com/tsuru/tsuru/...``?

**Comment by lucasweiblen** [comment]
yes, I run just godep pdate github,com/tsuru/tsuru/...

what a mess heheeh :/
lets see if we can solve this tomorrow

2015-03-16 23:45 GMT-03:00 Francisco Souza <notifications@github.com>:

**Comment by lucasweiblen** [comment]
all green.. don't believe \m/
sorry for the mess.. first time dealing with Godeps!

**Comment by fsouza** [comment]
No problem! It's really confusing x)

Thank you!

**Pull request merged**